ironfish/src/account/accounts.ts

```diff
@@ -588,18 +591,27 @@ export class Accounts {
     transactionFee: bigint,
     memo: string,
     receiverPublicAddress: string,
+    expirationSequence?: number | null,
   ): Promise<Transaction> {
     const heaviestHead = this.chain.head
     if (heaviestHead === null) {
       throw new ValidationError('You must have a genesis block to create a transaction')
     }
 
+    expirationSequence =
+      expirationSequence ??
+      heaviestHead.sequence + DEFAULT_TRANSACTION_EXPIRATION_SEQUENCE_DELTA
+    if (0 < expirationSequence && expirationSequence <= heaviestHead.sequence) {
```

Comment:
You can pass in a negative number for expiration here and it will continue. How about....?

```suggestion
    if (expirationSequence !== 0 && expirationSequence <= heaviestHead.sequence) {
```